src/components/button-toggle/button-toggle-group/button-toggle-group.stories.mdx

```diff
@@ -174,6 +166,36 @@ When the `fullWidth` prop is `true`, the buttons will expand in size to take up
   </Story>
 </Canvas>
 
+### Allowing deselection
+
+By default, a `ButtonToggleGroup` there is no way to deselect the currently-selected toggle button except by selecting another one.
```

Comment:
Think the first sentence is missing a word (in a ButtonToggleGroup?) or needs slightly tweaking